src/contribute/release-guide.md

```diff
@@ -511,7 +512,208 @@ If there are no issues, reply on the vote thread to close the voting. Then, tall
+    sha512sum -c apache-beam-2.5.0-source-release.zip.sha512
+    ```
+  * Build SDK
+  
+    ```
+    sudo apt-get install unzip
+    unzip apache-beam-2.5.0-source-release.zip
+    ```
+  * Setup virtualenv
+    
+    ```
+    pip install --upgrade pip
+    pip install --upgrade setuptools
+    pip install --upgrade virtualenv
+    virtualenv beam_env
+     . beam_env/bin/activate
+    ```
+  * Install SDK
+    
+    ```
+    pip install dist/apache-beam-2.5.0.tar.gz
+    pip install dist/apache-beam-2.5.0.tar.gz[gcp]
+    ```
+  * Setup GCP
+    
+    Please repeat following steps for every following test.
+    
+    ```
+    bq rm -rf --project=${YOUR_PROJECT} ${USER}_test
+    bq mk --project=${YOUR_PROJECT} ${USER}_test
+    gsutil rm -rf ${YOUR_GS_STORAGE]
+    gsutil mb -p ${YOUR_PROJECT} ${YOUR_GS_STORAGE}
+    gcloud alpha pubsub topics create --project=${YOUR_PROJECT} ${YOUR_PUBSUB_TOPIC}
+    ```
+    Setup your service account as described in ```Java Mobile Game``` section above.
+  
+    Produce data by using java injector:
+    * Configure your ~/.m2/settings.xml as following:
+      ```
+      <settings>
+        <profiles>
+          <profile>
+            <id>release-repo</id>
+            <activation>
+              <activeByDefault>true</activeByDefault>
+            </activation>
+            <repositories>
+              <repository>
+                <id>Release 2.4.0 RC3</id>
+                <name>Release 2.4.0 RC3</name>
+                <url>https://repository.apache.org/content/repositories/orgapachebeam-1031/</url>
+              </repository>
+            </repositories>
+          </profile>
+        </profiles>
+      </settings>
+      ```
+      _Note_: You can found the latest  ```id```, ```name``` and ```url``` for one RC in the vote email thread sent out by Release Manager.
+    
+    * Run 
+      ```
+      mvn archetype:generate \
+            -DarchetypeGroupId=org.apache.beam \
+            -DarchetypeArtifactId=beam-sdks-java-maven-archetypes-examples \
+            -DarchetypeVersion=${RELEASE_VERSION} \
+            -DgroupId=org.example \
+            -DartifactId=word-count-beam \
+            -Dversion="0.1" \
+            -Dpackage=org.apache.beam.examples \
+            -DinteractiveMode=false
+            -DarchetypeCatalog=internal
+            
+      mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.complete.game.injector.Injector \
+        -Dexec.args="${YOUR_PROJECT} ${YOUR_PUBSUB_TOPIC} none"
+      ```
+  * Run Leaderboard with Direct Runner
+    ```
+    python -m apache_beam.examples.complete.game.leader_board \
+    --project=${YOUR_PROJECT} \
+    --topic projects/${YOUR_PROJECT}/topics/${YOUR_PUBSUB_TOPIC} \
+    --dataset ${USER}_test
+    ```
+    Inspect results:
+    * Check whether there is any error messages in console.
+    * Goto your BigQuery console and check whether your ${USER}_test has leader_board table.
```

Comment:
I think both leader board and game stats should produce 2 tables: 1 for user and 1 for team. Could you mention that.

Should we include any example rows to look for inside the tables?

Reply:
Please review the latest commit https://github.com/apache/beam-site/pull/468/commits/3a1852ab0a9913cda76e246a6c9ffcb4b4728e8f .